webapp/src/pages/UserPage/UserPage.tsx

```diff
@@ -54,6 +107,21 @@ const UserComponentTemplate = ({
     );
   }
 
+  const unblockUserButton = (
+    <Button variant={ButtonVariant.SUBMIT} onClick={unblockUser}>
+      Unblock
+    </Button>
+  );
+
+  const blockUserButton = (
+    <Button variant={ButtonVariant.WARNING} onClick={blockUser}>
+      Block
+    </Button>
+  );
```

Comment:
I haven't run it yet, and I haven't seen it, but wouldn't it have been better to use the toggle switch?

Reply:
The toggle switch it's one option :+1:. I will give it a try.

Reply:
Done